modules/apps/object/object-web/src/main/resources/META-INF/resources/js/index.d.ts

```diff
@@ -3,65 +3,50 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
  */
 
-type LocalizedValue<T> = Liferay.Language.LocalizedValue<T>;
-
-interface Folder {
-	actions: {};
-	dateCreated: string;
-	dateModified: string;
+interface HTTPMethod {
+	href: string;
+	method: string;
+}
+interface Actions {
+	delete: HTTPMethod;
+	get: HTTPMethod;
+	permissions: HTTPMethod;
+	update: HTTPMethod;
+}
+interface AddObjectEntryDefinitions {
 	externalReferenceCode: string;
 	id: number;
-	label: LocalizedValue<string>;
-	name: string;
+	label: string;
+	related?: boolean;
+	system?: boolean;
 }
 
-type NotificationTemplate = {
-	attachmentObjectFieldIds: string[] | number[];
-	bcc: string;
-	body: LocalizedValue<string>;
-	cc: string;
-	description: string;
-	externalReferenceCode: string;
-	from: string;
-	fromName: LocalizedValue<string>;
-	id: number;
-	name: string;
-	objectDefinitionId: number | null;
-	subject: LocalizedValue<string>;
-	to: LocalizedValue<string>;
-	type: 'email' | 'userNotification';
-};
+interface LabelValueObject {
+	label: string;
+	value: string;
+}
 
-interface ObjectAction {
-	active: boolean;
-	conditionExpression?: string;
-	description?: string;
-	errorMessage: LocalizedValue<string>;
-	id?: number;
-	label: LocalizedValue<string>;
+interface IItem extends LabelValueObject {
+	checked?: boolean;
+}
+
+interface LabelNameObject {
+	label: string;
 	name: string;
-	objectActionExecutorKey: string;
-	objectActionTriggerKey: string;
-	objectDefinitionId?: number;
-	objectDefinitionsRelationshipsURL: string;
-	parameters: ObjectActionParameters;
-	script?: string;
 }
 
-interface ObjectActionParameters {
-	lineCount?: number;
-	notificationTemplateExternalReferenceCode?: string;
-	notificationTemplateId?: number;
-	objectDefinitionExternalReferenceCode?: string;
-	objectDefinitionId?: number;
-	predefinedValues?: PredefinedValue[];
-	relatedObjectEntries?: boolean;
-	script?: string;
-	secret?: string;
-	system?: boolean;
-	url?: string;
+interface LabelTypeObject {
+	label: string;
+	value: string;
```

Comment:
@igor-franca This should be `type` instead of `value`.

Reply:
@igor-franca Could you please check the sort of the types and interfaces in this file as well? I saw some types and interfaces are not sorted.

Reply:
This modification occurs in the commit "Create Empty Node" I will check with @alinedoleron  the reason for the change.

Reply:
I've verified that this change is related to an override in the stores population in `view_model_builder.jsp`.

Now we are using a [function](https://github.com/liferay/liferay-portal/blob/f00117d93cc8498a1aa5f7889e3afbe9a66e42d2/modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/ObjectDefinitionsDetailsDisplayContext.java#L145-L156) from another displayContext that assembles the storages with the label and value properties.

Reply:
Hi @igor-franca @alinedoleron , in that case, we should use the other interface called LabelValueObject, not change the LabelTypeObject interface.

Reply:
@igor-franca  Also the last three types in this file are not sorted and we have some places where we need to add a blank line.